packages/redux/src/raffles/actions/factories/fetchRaffleParticipationsFactory.ts

```diff
@@ -0,0 +1,43 @@
+import * as actionTypes from '@farfetch/blackout-redux/src/raffles/actionTypes';
+import { Config, Raffle, toBlackoutError } from '@farfetch/blackout-client';
+import { normalize } from 'normalizr';
+import raffleParticipationSchema from '../../../entities/schemas/raffleParticipations';
+import type { Dispatch } from 'redux';
+import type { GetRaffleParticipation } from '@farfetch/blackout-client/src/raffles/types/getRaffleParticipation.types';
```

Comment:
Import from the root of the package, please.

Reply:
done